lnd_services.go

```diff
@@ -78,22 +80,27 @@ type LndServicesConfig struct {
 
 	// MacaroonDir is the directory where all lnd macaroons can be found.
 	// Either this, CustomMacaroonPath, or CustomMacaroonHex should be set,
-	// but only one of them, depending on your macaroon preferences.
+	// but only one of them, depending on macaroon preferences.
```

Comment:
nit: squash these white space/wording changes with the first commit so only the TLS changes are in this commit.